gosdkexample/filtergvk/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework/gosdk"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework/gosdk/command"
+)
+
+// In this example, we use set a field in resources selecting by GVK.
+
+func main() {
+	cmd := command.AsMain(context.TODO(), mutateByGVK, command.Options{})
```

Comment:
Do I need to use the command here?  What's the advantage of it?  Can I just have a regular old `main`?

Reply:
The only advantage is that command can ship with help text. But that's not a huge advantage.
I have updated the examples